app/static/styles/newacct.css

```diff
@@ -0,0 +1,11 @@
+body {background-color: white;}
+
+.myDiv {
```

Comment:
CSS class names should be descriptive. That is, use meaningful or generic class names. See the Google HTML/CSS Style Guide (https://google.github.io/styleguide/htmlcssguide.html#CSS_Style_Rules) for more information and recommendations on CSS styling